lib/phoenix/tracker/replica.ex

```diff
@@ -77,9 +77,13 @@ defmodule Phoenix.Tracker.Replica do
     %Replica{replica | last_heartbeat_at: now_ms()}
   end
 
-  defp now_ms, do: System.system_time(:milli_seconds)
+  @otp_vsn :erlang.system_info(:otp_release) |> :erlang.list_to_integer()
+  @time_unit_ms if @otp_vsn < 19, do: :milli_seconds, else: :milliseconds
+  @time_unis_us if @otp_vsn < 19, do: :micro_seconds, else: :microseconds
```

Comment:
> Is this necessary? I believe Elixir should take care of this within System, no?

Yes, it takes care of it and translates `:milli_seconds` to `:milliseconds`, but OTP 18 has different typespecs for time units than OTP 19 (http://devdocs.io/erlang~18/erts-7.3/doc/html/erlang#type-time_unit), hence this small hack